cmd/nvidia-mig-manager/main.go

```diff
@@ -418,7 +418,7 @@ func migReconfigure(ctx context.Context, migConfigValue string, clientset *kuber
 		return fmt.Errorf("error creating reconfigure instance: %w", err)
 	}
 
-	return rcfg.Run()
+	return rcfg.Run(ctx)
```

Comment:
we already `ctx` in the `rcfg, err := reconfigure.New(ctx, clientset, migPartedBinary, opts)` line. Can we access the context from the struct itself?

Reply:
Updated.